addons/website/static/src/snippets/s_cta_badge/000.scss

```diff
@@ -0,0 +1,30 @@
+.s_cta_badge {
+    --s_cta_badge_link_margin: 20px;
+
+    margin: 1rem;
```

Comment:
Is there a reason you don't set this (L4) and L7 in the xml ? 

Also, having a forced margin on the badge makes some layout unachievable. Ex: 

<img width="897" alt="Screenshot 2024-08-05 at 18 01 39" src="https://github.com/user-attachments/assets/57ba1f43-cf88-41ca-85d7-9abc0ff2c876">

I know it's not ideal and I'd better have a controller on margins, but IMO it will be more flexible to remove the horizontal margin, as the user will still be able to use text spaces to add some spacing between elements.

Otherwise LGTM 👍

Reply:
Yes I remember we discussed it 😅 Let's remove the left and right margin.

For the rest there is no specific reason, no. I was very careful about putting classes in the XML as I'm not sure what can be put there or not but I'll add these there

Reply:
The rest of the CSS made sense to me because the user can remove elements and we need to keep the rules if he adds them back...

But actually, the margins you set L20 and L27 can be transformed to white spaces aswell, no ?